bots/github/__init__.py

```diff
@@ -300,30 +300,40 @@ class Checklist(object):
     def __init__(self, body=None):
         self.process(body or "")
 
+    def format(self, item, check):
```

Comment:
This is a function of `Checklist` but operates on a single item - can we rename it to something like `format_item` and/or add a comment?